src/models/FaqModel.js

```diff
@@ -0,0 +1,15 @@
+import is from "ramda/src/is";
+
+export default class Faq {
+  constructor({
+    de_question = null,
```

Comment:
Spannende Frage: Hier sind die `de` und `en` Varianten explizit angegeben.`
Wie verhält sich das mit der dynamischen Auswahl in [FAQ.vue](https://github.com/ClockGU/clock-frontend/blob/295e4916a0e7a9bdba54c58959b30157390cede7/src/views/FAQ.vue#L56-L63)?

Oder anders: _wenn_ wir im Backend eine neue Sprache ergänzen, müssen Model und Service auch angepasst werden? 

Ist jetzt kein Ding, es ist im Moment eh unwahrscheinlich.

Reply:
Aber es wäre ein Überlegung, ob beim nötigen Umbau des Backends (wegen public Permission) gleich der Endpoint so gebaut wird, dass er die locale annimmt und nur die angefragte Sprache zurückgibt (siehe anderer Kommentar).

Reply:
Ich kann ja mal ein Issue fürs Backend schreiben, dass das Umsetzen würde